cmd/ctr/commands/tasks/delete.go

```diff
@@ -72,14 +71,32 @@ var deleteCommand = cli.Command{
 				return cli.NewExitError("", int(ec))
 			}
 		} else {
-			status, err := task.Delete(ctx, opts...)
-			if err != nil {
-				return err
-			}
-			if ec := status.ExitCode(); ec != 0 {
-				return cli.NewExitError("", int(ec))
+			for _, target := range context.Args() {
+				task, err := loadTask(ctx, client, target)
+				if err != nil {
+					return err
+				}
+				status, err := task.Delete(ctx, opts...)
+				if err != nil {
+					return err
+				}
+				if ec := status.ExitCode(); ec != 0 {
+					return cli.NewExitError("", int(ec))
```

Comment:
This can early return before attempting to delete all tasks; seems like you need a way to store (and report) non-zero exit codes if you are going to do multi-delete.

Reply:
will fix it